app/src/main/kotlin/com/bitwarden/authenticator/ui/authenticator/feature/search/ItemSearchViewModel.kt

```diff
@@ -234,12 +239,12 @@ data class ItemSearchState(
     data class DisplayItem(
         val id: String,
         val authCode: String,
-        val issuer: String?,
+        val title: String,
+        val subtitle: String? = null,
```

Comment:
Second UI model that I updated to use `title` and `subtitle`. This one is only for search results.